src/main/java/uk/gov/hmcts/sscs/email/EmailAttachment.java

```diff
@@ -55,4 +56,24 @@ public String toString() {
         return String.format("EmailAttachment{contentType='%s', filename='%s'}",
                 filename, contentType);
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
cannot you use lombok as well here -> https://projectlombok.org/features/EqualsAndHashCode